components/vis/globe-cesium/component.js

```diff
@@ -145,6 +145,7 @@ class GlobeCesiumComponent extends PureComponent {
   }
 
   componentWillReceiveProps(nextProps) {
+    console.log('this.props.layerActive', this.props.layerActive, 'nextProps.layerActive', nextProps.layerActive);
```

Comment:
Remove it

Reply:
👍
I keep forgetting `console.log` statements here and there... sorry for that! 😅